recipes/r-minems2/meta.yaml

```diff
@@ -18,19 +18,19 @@ requirements:
   build:
     - {{ compiler('cxx') }}
   host:
-    - r-base
-    - r-rcpp
-    - bioconductor-msnbase
-    - r-stringr
-    - r-igraph
-    - r-matrix
+    - r-base>=3.5.1
+    - r-rcpp=0.12.18
+    - bioconductor-msnbase=2.8.3
+    - r-stringr=1.3.1
```

Comment:
Are the version requirements for CRAN packages really that exact? I suspect you want a minimum version instead.

Reply:
I'd like to. But `bioconductor-msnbase` is imposing to have R 3.5.1. I think I'll try to upgrade `bioconductor-msnbase` first, if I succeed I will be able to remove all versions requirements.
In the meantime I'll put that pullrequest aside.

Reply:
The msnbase rebuild should be happening any time now, so maybe the best plan is to hold off a bit.

Reply:
I've removed all the fixed version requirements. The checks pass, so I suppose new MSnbase build has been made available.